libs/converter-nh-accuvote/src/convert/convert_election_definition.ts

```diff
@@ -43,16 +37,37 @@ export function convertElectionDefinition(
     let success = true;
     const issues = [...headerIssues];
 
+    const findLayoutResult = findLayout([
+      cardDefinition.front,
+      cardDefinition.back,
+    ]);
+
+    if (findLayoutResult.isErr()) {
+      return err({
+        issues: [
+          ...issues,
+          typedAs<ConvertIssue>({
+            kind: ConvertIssueKind.TimingMarkDetectionFailed,
+            message: 'failed to detect timing marks',
+            side: 'front',
+          }),
+        ],
+      });
+    }
+
+    let [frontLayout, backLayout] = findLayoutResult.ok().layouts;
+
+    if (
+      frontLayout.grid.metadata.side === 'back' &&
+      backLayout.grid.metadata.side === 'front'
+    ) {
+      [frontLayout, backLayout] = [backLayout, frontLayout];
+    }
+
     let paperSize = election.ballotLayout?.paperSize;
 
-    const frontExpectedPaperSize = getTemplateBallotPaperSize({
-      width: cardDefinition.front.width,
-      height: cardDefinition.front.height,
-    });
-    const backExpectedPaperSize = getTemplateBallotPaperSize({
-      width: cardDefinition.back.width,
-      height: cardDefinition.back.height,
-    });
+    const frontExpectedPaperSize = frontLayout.grid.geometry.ballotPaperSize;
+    const backExpectedPaperSize = backLayout.grid.geometry.ballotPaperSize;
```

Comment:
Is it possible for these to be undefined or for them to not match, or is that checked in the interpreter now? Thinking it might be possible to simplify the next block below

Reply:
They can no longer be undefined nor can the front and back not match, though they could mismatch the expected size.